src/backend/tests/it/utils/pocketic.rs

```diff
@@ -108,6 +108,7 @@ pub(crate) fn init_arg() -> Arg {
             issuer_origin: ISSUER_ORIGIN.to_string(),
             credential_type: CredentialType::ProofOfUniqueness,
         }]),
+        api: None,
```

Comment:
@bitdivine: Will you add some integration tests that exercise the API disabling?

Reply:
Disabling APIs matters in the migration, so I was going to test that in the migration e2e tests.  After the migration is complete I expect all of this will be deleted again. :smile: